res/skins/Deere/skin.xml

```diff
@@ -8,39 +8,42 @@
     <language>en</language>
     <license>Creative Commons Attribution, Share-Alike 3.0 Unported</license>
     <attributes>
+      <!-- Set skin defaults. -->
       <attribute config_key="[Master],num_decks">4</attribute>
       <attribute config_key="[Master],num_samplers">16</attribute>
       <attribute config_key="[Master],num_preview_decks">1</attribute>
-      <attribute config_key="[Library],show_library">1</attribute>
```

Comment:
Bring this back and use `[Skin],show_library`. I tried starting Mixxx with a new settings directory and Deere did not show the library.

Reply:
added back